ixmp/backend/jdbc.py

```diff
@@ -226,6 +238,49 @@ def get_log_level(self):
         levels = {v: k for k, v in LOG_LEVELS.items()}
         return levels.get(self.jobj.getLogLevel(), 'UNKNOWN')
 
+    def set_doc(self, domain, docs):
+        """ Save documentation to database
```

Comment:
is this consistent with space after `"""` or without?

Reply:
updated